percona-server-mongodb-4.4/Dockerfile

```diff
@@ -0,0 +1,120 @@
+FROM centos:7
```

Comment:
maybe we need to use centos:8 ... @EvgeniyPatlan  what do you think ?

Reply:
yeah. maybe let's do for centos:8